### PR TITLE
Creating class with common functionality in AEM and Jahia for templat…

### PR DESCRIPTION
**Pull request by josecarlosgt**
…e content model implementation.

This class contains common functionality (properties and methods) found in AEM and Jahia template content model implementations.

### CODE REVIEW

**Comment by josecarlosgt** [comment]
The BND plugin is left as follows:

- com.github.jknack.handlebars dependency is exported by the Core project. Thus, AEM and Jahia projects do not need to embed it. The reason for this is that it is not possible to embed handlebars  in all Core, AEM and Jahia projects. Attempting to doing so raises a LinkageErrorError exception as the server tries to load the same class twice.

- Give that both AEM and Jahia projects will use handlebars classes exported by Core, all handlebars dependencies: mozilla.javascript, antlr4-runtime, commons-lang3 are embedded only in Core and not in the AEM and Jahia projects anymore.

**Comment by jbarrera-xumak** [comment]
For bundles that are being embedded I prefer export them using the tag <_exportcontents> and not </Export-Package> as it is in the API bundle, to avoid conflicts.

**Comment by josecarlosgt** [comment]
Done in the previous commit. It builds successfully for both AEM and Jahia. I will run more tests tomorrow uninstalling the bundles.

**Comment by jbarrera-xumak** [comment]
Ok thanks, could you also do rebase form master and update the changelog file please?

**Comment by josecarlosgt** [comment]
Done. Commits in this branch were moved ahead and CHANGELOG file was updated in previous commit.

**Comment by josecarlosgt** [comment]
Last update to the CHANGELOG was re-written to make it standard with other updates.

File: src/main/java/danta/core/templating/AbstractTemplateContentModelImpl.java [hunk 1]
```diff
@@ -0,0 +1,299 @@
+/**
+ * Danta AEM Bundle
```
Comment (jbarrera-xumak) [inline src/main/java/danta/core/templating/AbstractTemplateContentModelImpl.java]: Hi @josecarlosgt, Could you fix the the bundle name, please?

**Comment by josecarlosgt** [comment]
Done!

File: src/main/java/danta/core/templating/AbstractTemplateContentModelImpl.java (hunk 1 above)
Comment (josecarlosgt) [reply src/main/java/danta/core/templating/AbstractTemplateContentModelImpl.java]: Done!

**Review by jbarrera-xumak — APPROVED** [review]
+1

**Pull request merged**